ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/block/BlockProcessorCapella.java

```diff
@@ -28,6 +47,9 @@
 import tech.pegasys.teku.spec.schemas.SchemaDefinitionsBellatrix;
 
 public class BlockProcessorCapella extends BlockProcessorBellatrix {
+  private static final Bytes ETH1_WITHDRAWAL_PREFIX = Bytes.fromHexString("0x01");
+  private static final Bytes WITHDRAWAL_BUFFER = Bytes.repeat((byte) 0x00, 11);
```

Comment:
this will likely move to capella helpers once it exists

Reply:
I'd have this pre-concatenated with the `ETH1_WITHDRAWAL_PREFIX` to save processing when we use it.  Buffer isn't the right name for it either since it's not actually a Buffer.  Maybe just `ETH1_WITHDRAWAL_PREFIX_AND_PADDING`?

Reply:
i'll just concatenate it into a const, and won't need that value separately at all

Reply:
We have this value available on `MiscHelpersCapella`.